ui/admin/app/styles/app.scss

```diff
@@ -953,9 +952,29 @@
 
 // workers
 .workers {
-  .rose-dropdown-content {
-    max-height: 300px;
-    overflow: scroll;
+  .tags-list {
+    display: flex;
+    margin-bottom: 1rem;
+
+    .toolbar-refresher {
+      margin-left: auto;
+    }
+
+    .grouping {
```

Comment:
I think the braces need to be reformatted here

Reply:
updated!

Reply:
I think the whitespace is still wonky 😅

Reply:
Yup looks good! Prettier would be the one that would fix this whitespace formatting instead of lint